File: scripts/inspect_orgs.js
const { createClient } = require("@supabase/supabase-js");

const url = process.env.SUPABASE_PROD_URL;
const key = process.env.SUPABASE_PROD_SERVICE_ROLE_KEY;

const supabase = createClient(url, key);

async function main() {
  const { data: allOrgs, error } = await supabase
    .from("organizations")
    .select("slug, name, location, municipality, province")
    .not("location", "is", null)
    .not("location", "eq", "")
    .is("municipality", null);

  if (error) {
    console.error(error);
    return;
  }

  console.log(
    `Found ${allOrgs.length} organizations with location set but municipality null`,
  );
  for (let i = 0; i < Math.min(20, allOrgs.length); i++) {
    const org = allOrgs[i];
    console.log(`- ${org.name} (${org.slug}): location="${org.location}"`);
  }
}

main();

File: scripts/inspect_orgs.py
import os
from supabase import create_client, Client

url = os.environ.get("SUPABASE_PROD_URL")
key = os.environ.get("SUPABASE_PROD_SERVICE_ROLE_KEY")

if not url or not key:
    print("Missing credentials")
    exit(1)

supabase: Client = create_client(url, key)

# Fetch Adamant
adamant = supabase.table("organizations").select("slug, name, location, municipality, province").eq("slug", "adamant").execute()
print("Adamant:", adamant.data)

# Fetch Accelerating Community Energy Transformation
acet = supabase.table("organizations").select("slug, name, location, municipality, province").eq("slug", "accelerating-community-energy-transformation").execute()
print("ACET:", acet.data)
